src/index.js

```diff
@@ -1604,6 +1630,30 @@ const initializeFormElements = () => {
     console.log(result);
   };
 
+  // Malicious Mint
+  maliciousContractInteractionButton.onclick = async () => {
+    let contractAddress;
+
+    if (networkName === 'mainnet') {
+      contractAddress = '0x000062Accd1a9d62eF428eC86cA3dD4f45120000';
+    } else {
+      contractAddress = '0x00008F1149168C1D2fa1eBa1Ad3e9cD644510000';
```

Comment:
nit: I would rather move this into the [sample contracts file](https://github.com/MetaMask/test-dapp/blob/main/src/onchain-sample-contracts.js#L33), to make it easier to locate and maintain in the future (but not blocking)

Reply:
done